docs/source/development/traffic_portal.rst

```diff
@@ -44,6 +44,32 @@ Installing The Traffic Portal Developer Environment
 #. Navigate to the ``traffic_portal`` subdirectory of your cloned repository.
 #. Run ``npm install`` to install application dependencies into ``traffic_portal/node_modules``. Only needs to be done the first time unless ``traffic_portal/package.json`` changes.
 #. Run ``bower install`` to install client-side dependencies into ``traffic_portal/app/bower_components``. Only needs to be done the first time unless ``traffic_portal/bower.json`` changes.
+#. Make sure that compass is installed and functioning correctly by running ``compass version``. If ``compass`` is not running, you can install it on macOSas follows:
+    #. macOS comes with its own version of ruby built into it. In order to install compass, if you run a command like ``sudo gem install compass``, it will take the default ruby available in mac. In order to properly install compass, here is what you should be doing:
+    #. ``brew install ruby``
+    #. ``gem update --system``
+    #. ``gem install compass``
+       Note: You might need to install sass before you install compass. You can do that by running:
```

Comment:
Missing blank lines; this causes rendering issues.

Also, this is a note, and so should be using the `.. note::` directive, or possibly the `.. tip::` directive if more appropriate

Also also, you made the step that this optional admonition contains a part of the overall process. So if this is truly optional, then the optional step should be contained in the admonition, and if it isn't then this admonition should not exist.

Reply:
This is not an optional step. I changed the text to reflect that.